README.md

```diff
@@ -1 +1,125 @@
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Leaderboard <a name="about-project"></a>
+
+The leaderboard website displays scores submitted by different players. It also allows you to submit your score. All data is preserved thanks to the external Leaderboard API service.
+
+## 🛠 Built With <a name="built-with"></a>
+
+<!-- ### Tech Stack <a name="tech-stack"></a> -->
+
+HTML, CSS, JavaScript
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+<!-- - **[Add TAsks]**
+- **[Edit TAsks]**
+- **[Mark tasks as completed]**
+- **[Delete a task]**
+- **[Delete all tasks]** -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Screenshoots
+
+<!-- ![App Screenshoot](/src/Screenshot_1.png)
+![App Screenshoot](/src/Screenshot_2.png)
+![App Screenshoot](/src/Screenshot_3.png) -->
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+<!-- - [Live Demo Link](https://auducodes-todo-app.netlify.app/) -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+Clone this repository to your machine and you are go to go:
+
+https://github.com/audukehinde/leaderboard.git
+
+### Prerequisites
+
+Web Browser
+
+### Setup
+
+Download the repo.
+
+### Install
+
+Install this project with:
+
+cd leaderboard
+
+npm install
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Kehinde Audu**
+
+- GitHub: [@githubhandle](https://github.com/audukehinde)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/kehinde-audu-a44926175/)
+- Twitter: [Twitter](@auducodes)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project kindly show you support
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] Kindly add a `Future Features` section, including 1-3 potential features to be added.
- [ ] The link to the MIT license is broken. Kindly fix it.